projects/ngrx.io/content/guide/store/selectors.md

```diff
@@ -294,6 +294,83 @@ export const selectFeatureCount = createSelector(
 );
 </code-example>
 
+## Using Signal Selector
+The `selectSignal` method expects a selector as an input argument and returns a signal of the selected state slice. It has a similar signature to the `select` method, but unlike `select`, `selectSignal` returns a signal instead of an observable.
+
+### Example Usage in Components
+```typescript
+import { Component, inject } from '@angular/core';
+import { NgFor } from '@angular/common';
+import { Store } from '@ngrx/store';
+
+import { selectUsers } from './users.selectors';
+
+@Component({
+  standalone: true,
+  imports: [NgFor],
+  template: `
+    <h1>Users</h1>
+    <ul>
+      <li *ngFor="let user of users()">
+        {{ user.name }}
+      </li>
+    </ul>
+  `
+})
+export class UsersComponent {
+  private readonly store = inject(Store);
+
+  // type: Signal<User[]>
+  readonly users = this.store.selectSignal(selectUsers);
+}
+```
+
+### ComponentStore Selectors
```

Comment:
This isn't required.
This page is for `@ngrx/store`, the component store documentation has already been updated in https://ngrx.io/guide/component-store/read#selectsignal-method